bundle/manifests/operator.ibm.com_commonservices.yaml

```
@@ -8,6 +8,7 @@ metadata:
     app.kubernetes.io/instance: ibm-common-service-operator
     app.kubernetes.io/managed-by: ibm-common-service-operator
     app.kubernetes.io/name: ibm-common-service-operator
+    foundationservices.cloudpak.ibm.com: commonservice
```

we dont use this label, can we use the same keycloak label we already have for all the keycloak CRs?
```suggestion
    foundationservices.cloudpak.ibm.com: keycloak
```

---

We cannot say that CommonSvcs CD is labelled with Keycloak, since it's not Keycloak :)

---

We'll need to add the value to our backup doc, I agree with Piotr that its not just keycloak.

---

we're not backing up Common services though, we're only backing up keycloak, this might imply it's backing up more than just Keycloak. I'll drop it though, semantic in the end.

---

If we want to support that, then we'd need something like:
`foundationservices.cloudpak.ibm.com/keycloak: backup`

Otherwise if you wanted to backup another common services component you wouldn't be able to add a second value to the label.

It would have been good to have `foundationservices.cloudpak.ibm.com/component: commonservice` but we missed that for the keycloak label.

---

We should probably have done a design session on the labels sooner :)

---

This is just what CS did for their own taxonomy. They wanted us to follow their convention not ours. I think we should follow the taxonomy of the product but here we are :)